storybook/pages/SwapModalPage.qml

```diff
@@ -145,35 +145,6 @@ SplitView {
                 destroyOnClose: true
                 swapInputParamsForm: adaptor.swapFormData
                 swapAdaptor: adaptor
-                plainTokensBySymbolModel: ListModel {
```

Comment:
Why removing this? I see that you added it to the other model but this is here to test the concatenation of 2 different models (sources)